apps/storefront/src/pages/QuickOrder/components/QuickAdd.tsx

```diff
@@ -335,7 +335,9 @@ export default function QuickAdd(props: AddToListContentProps) {
 
     const productsToValidate = mapCatalogToValidationPayload(variantInfoList, skuValue);
 
-    const { validProducts, errors } = await validateProducts(productsToValidate);
+    // When adding products to a cart, warnings need to be treated as errors.
+    // We are doing this in the frontend for now, but it will be moved to the backend in the future.
+    const { validProducts, errors } = await validateProducts(productsToValidate, 'cart');
```

Comment:
Question, we can also add products to the cart from the purchased products table, do we need to also pass the "cart" prop to this function?